### ISSUE TITLE
Requirements file

### ISSUE THREAD

**Issue by zhongxiangboy** [darsh10/split_encoder_pointer_summarizer#1]
In order to enable running code, can you provide a **requirement description**?

**Comment by darsh10** [comment]
Yes, we have it in the description. Please follow https://github.com/atulkum/pointer_summarizer.
